File: medium/array/Two Sum II - Input Array Is Sorted.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int l = 0;
        int i = nums.length - 1;
        while(i>l){
            if(nums[i]+nums[l]==target){
                return new int[]{l+1,i+1};
            }else if(nums[i]+nums[l]>target){
                i--;
            }else{
                l++;
            }
        }return new int[]{-1,-1};
    }
}

File: medium/array/Two Sum II - Input Array Is Sorted.py
class Solution(object):
    def twoSum(self, nums, target):
        i = len(nums)-1
        l = 0
        while i>l:
            if nums[i] + nums[l] == target:
                return [l+1,i+1]
            elif nums[i]+nums[l]>target:
                i-=1
            else:
                l+=1
        return [-1,-1]